.github/workflows/openzeppelin_test_11.yml

```diff
@@ -94,7 +94,7 @@ jobs:
         run: |
           cargo build
           rm -rf ./devtools/chain/data
-          ./target/debug/axon --config devtools/chain/config.toml --genesis devtools/chain/genesis_single_node.json > /tmp/log 2>&1 &
+          ./target/debug/axon --config devtools/chain/config.toml --genesis devtools/chain/genesis_single_node.json run > /tmp/log 2>&1 &
```

Comment:
I kinda think it should be `axon run --config ... --genesis ...`.

We may have in the future commands like `axon init`, `axon gen-key` which wouldn't need (all) these args.